test/e2e/storage/subpath.go

```diff
@@ -718,6 +785,15 @@ func (s *gcepdSource) createVolume(f *framework.Framework) volInfo {
 	}
 }
 
+func (s *gcepdSource) readOnlyVolume() *v1.VolumeSource {
```

Comment:
these functions are "creating" a readOnlyVolume right? The naming is a little ambiguous, at first I thought it was an "isReadOnlyVolume" type function but it doesn't seem like that.

Reply:
createReadOnlyVolumeSource?